util/cuda/cudaRGB.cu

```diff
@@ -42,23 +43,58 @@ __global__ void RGBToRGBAf(uchar3* srcImage,
         return;
 
 //	printf("cuda thread %i %i  %i %i pixel %i \n", x, y, width, height, pixel);
-		
-	const float  s  = 1.0f;
-	const uchar3 px = srcImage[pixel];
-	
-	dstImage[pixel] = make_float4(px.x * s, px.y * s, px.z * s, 255.0f * s);

```

Comment:
The file was indented with spaces except for these 5 lines which were tabbed.  Made it consistent.